src/test/java/seedu/address/logic/commands/ClearFacilitiesCommandTest.java

```diff
@@ -0,0 +1,31 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalAddressBook.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+public class ClearFacilitiesCommandTest {
+
+    @Test
+    public void execute_emptySportsPA_success() {
+        Model model = new ModelManager();
+        Model expectedModel = new ModelManager();
+        assertCommandSuccess(
+                new ClearFacilitiesCommand(), model, ClearFacilitiesCommand.MESSAGE_SUCCESS, expectedModel);
+    }
+
+    @Test
+    public void execute_nonEmptySportsPA_success() {
+        Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
```

Comment:
```suggestion
        Model expectedModel = new ModelManager(getTypicalAddressBookEmptyFacilityList, new UserPrefs());
```
Maybe we could do this so we can check that only the facility list has been cleared?